openedx_webhooks/github/dispatcher/rules/github_activity.py

```diff
@@ -38,7 +38,8 @@ def _is_sender_edx_user(event):
         bool
     """
     try:
-        return event.sender.is_edx_user
+        event.sender
+        return True
```

Comment:
This seems like a strange way to do it: depending on event.sender to raise NotFoundError?  As a property, it feels more natural for it to return None in this case.